flank-scripts/build.gradle.kts

```diff
@@ -156,49 +157,47 @@ tasks.register("download") {
     ant.invokeMethod("get", mapOf("src" to sourceUrl, "dest" to destinationFile))
 }
 
-tasks.register("checkIfVersionUpdated") {
-    group = "verification"
-    val isVersionChanged = withTempFile {
-        outputStream().use {
+val checkIfVersionUpdated by tasks.registering(Exec::class) {
+    commandLine("git", "fetch", "--no-tags")
 
-            project.exec {
-                commandLine("git", "fetch", "--no-tags", "-v")
-            }
+    doLast {
+        val changedFiles = execAndGetStdout("git", "diff", "origin/master", "HEAD", "--name-only").split("\n") +
```

Comment:
Isn't `git diff origin/master --name-only` enough? I mean that includes `diff` with `HEAD`

Reply:
`git diff origin/master --name-only ` compare with local changes

Reply:
So I am wrong in the end 😄 thanks!